### ISSUE TITLE
[Active User - Add User]: Screen reader does not announce state as expanded on 'Select location' combo box when activate through enter key.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13203]
### Steps to reproduce:
1.	Open URL and login with valid credentials.
2.	Navigate to 'Dashboard view' by tab key and activate through enter key.
3.	Navigate to 'Navigation menu-> User-> Active user' and activate through enter key.
4.	Navigate to Add a user control and activate through enter key.
5.	Fill all required details and click 'Next'.
6.	Verify that screen reader does announce state as expanded on Select location combo box when activate through enter key or not.

### Actual behavior:
Screen reader does not announce state as expanded on Select location combo box when activate through enter key.

### Expected behavior:
Screen reader should announce state as expanded on Select location combo box when activate through enter key.

### Additional Notes:
1.	Same issue repro on 'Active teams & Groups-> Edit email addresses->Aliases domain and More actions list'.
2.	Same issue repro on Change view icon control in 'Teams & groups-> Deleted groups' page.
3.	Same issue repro on 'Filter' control in Roles-> Role assignments-> Filter.
4.	Same issue repro on 'Filter' control in Service Health->Incident--->Filter. 

### System configuration
Application: M365 Admin Portal
URL: https://admin.microsoft.com/Adminportal/Home
Browser: Edge, Version 96.0.1054.62 (Official build) (64-bit)

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA Version 2021.1

#### Windows version:
Window 11, Version 21H2 (OS Build 22000.376)

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, also reproducible through NVDA version 2020.4

**Comment by feerrenrut** [comment]
This needs further investigation by Microsoft developers. Please provide a minimal test case for us to investigate. I'm closing this issue for now.
